BSM/Chapter_04_01_Logistic_regression_for_NBA_clutch_free_throws.ipynb

```diff
@@ -0,0 +1,426 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pymc3 as pm\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import arviz as az"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "az.style.use('arviz-white')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Logistic regression for NBA clutch free throws\n",
+    "\n",
+    "### Chapter 4.3.3: Generalized linear models\n",
+    "\n",
+    "The NBA clutch free throws data set has three variables for player $i=1,...,10$:\n",
+    "\n",
+    "1.  $Y_i$ is the number clutch free throws made\n",
+    "2.  $N_i$ is the number clutch free throws attempted\n",
+    "3.  $q_i$ is the proportion of the non-clutch free throws made\n",
+    "\n",
+    "We model these data as\n",
+    "$$Y_i\\sim\\mbox{Binomial}(N_i,p_i),$$\n",
+    "where $p_i$ is the true probability of making a clutch shot.  The objective is to explore the relationship between clutch and overall percentages, $p_i$ and $q_i$.  We do this using two logistic regression models:\n",
+    "\n",
+    "1.  $\\mbox{logit}(p_i) = \\beta_1 + \\beta_2\\mbox{logit}(q_i)$\n",
```

Comment:
I do not have book in front of me but am curious. Why do we need two models to explore relationship between q and p. Why isn't the first model enough? Once we have beta_2 it's should be clear what the relationship is?